packages/turtle-language-server/package.yaml

```
@@ -0,0 +1,16 @@
+---
+name: turtle-language-server
+description: A language server (by Stardog) for Turtle
+homepage: https://www.npmjs.com/package/turtle-language-server
+licenses:
+  - Apache-2.0
+languages:
+  - turtle
```

```suggestion
  - Turtle
```

Language should be capitalized